index.js

```diff
@@ -71,6 +72,7 @@ function ddbtest(test, projectName, tableDef, region) {
     dynalite.listen(4567, function(err) {
       if (err) throw err;
       listening = true;
+      test.onFinish(function(){ dynalite.close(); });
```

Comment:
Does this cause any problems if dynalite is already closed?

Reply:
Nope, not in my tests.